### PR TITLE
Project 2 : Responsive Web Design Milestone 1 

### PR DESCRIPTION
**Pull request by mariordgez**
### ****Code review Milestone 1: Responsive web design****
This is the seconf project of the Microverse technical curriculum, the objective is to create a web page that resembles one from a TNW article, and to make the adapt to different widths utilizing media queries.
### _Milestone 1 
In the first milestone, you will prepare the tools to develop this website and will create 4 sections.

    Set up a new the repository

    Prepare the main HTML file

    Prepare your project to work with 3 screen sizes: mobile, tablet, and desktop using mediaqueries. What are the 2 breakpoints?

    Create these 4 sections of the website for the 3 screen sizes (see below)
  
### HEADER
imagen
imagen
imagen

### FOOTER
imagen

### HIGHLIGHTS
imagen

### LATEST FUND
imagen
imagen

### Workflow
We actually didn't work according to the Microverse order of implementation and decided to work on the sections starting from the top of the page to the bottom, and we first finished the mobile version of the page

### Conclusion

I learned a lot of tools and feel like I earned lots of experience just from this project alone, I hope to keep improving and keep getting better at this, thanks to the reviewers for your time, I will apply any correction or improvement as soon as possible, have a nice day!.

### CODE REVIEW

**Review by Elbie-em — COMMENTED** [review]
## Status: Invalid Submission :red_circle:

Hello @mariordgez  and @Edungit, you have done a great job with your README file thus far.
However, this code review request will be marked as invalid in your dashboard due to the following reasons:

-  You did not follow the correct [GitFlow](https://microverse.zendesk.com/hc/en-us/articles/360055660894-GitHub-Flow) required for this project. You should have a development branch as default, and make a pull request from the milestone_1 branch into the development branch.

- Your pull request should have a summary describing the tasks completed for this review.

Please make the necessary changes and submit another code review request once you are ready! 🙂.

> If you have any queries you can contact me on slack @ Elbie Moonga'

### :exclamation: NOTE you still have 3 code reviews left for this project as it is marked as invalid. Once you submit a valid code review request, you will have have 2 code reviews left for this project as described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,805 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/cf2fdf395d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <article>
```
Comment (henatan99) [inline index.html]: This 'article' tag is not good here, please replace it with a 'div' tag which can be served as a container element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+        <!--Nav-bar Section-->
+        <nav class="navbarDesktop">
```
Comment (henatan99) [inline index.html]: Change this nav tag to header-tag and the child div elements to nav

File: index.html [hunk 3]
```diff
@@ -0,0 +1,805 @@
...
+            <div class="blackNav">
```
Comment (henatan99) [inline index.html]: You should replace this div tag with a nav tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,805 @@
...
+                    <li class="blackitem"><a href="#">Spaces</a></li>
+                    <li class="blackitem"><a href="#">Terms & Conditions</a></li>
+                    <li class="blankSpace"></li>
+                    <li class="blackitem"><a href="#">About</a></li>
+                    <li class="blackitem"><a href="#">Advertise</a></li>
+                    <li class="blackitem"><a href="#">Jobs</a></li>
+                    <li class="blackitem"><a href="#">Contact</a></li>    
+                </ul>
+            </div>
+            <div class="whiteNav">
```
Comment (henatan99) [inline index.html]: Replace this div tag with nav tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,805 @@
...
+                <ul>
+                    <li class="whiteitem"><a href="#">LATEST</a></li>
+                    <li class="whiteitem"><a href="#">HARD WORK</a></li>
+                    <li class="whiteitem"><a href="#">PLUGGED</a></li>
+                    <li class="whiteitem"><a href="#">README</a></li>
+                    <li class="whiteitem"><a href="#">GROWTH QUARTERS</a></li>
+                    <li class="blankSpace"></li>
+                    <li class="search"><div class="searchicon"><i class="fa fa-search" aria-hidden="true"></i></div></li>
```
Comment (henatan99) [inline index.html]: The search icon should also have a-tag for link

File: index.html [hunk 6]
```diff
@@ -0,0 +1,805 @@
...
+
+        <nav class="navbarMobile">
+            <div class="logo">
+                <a href="#Top">
+                    <img src="Images/tnw.PNG" alt="tnw">
+                </a>
+            </div>
+            <div class="white-space"></div>
+               <div class="search"> <i class="fa fa-search" aria-hidden="true"></i> </div>
+               <div class="menu-bar"> <i class="fa fa-bars" aria-hidden="true"></i> </div>
```
Comment (henatan99) [inline index.html]: Both search icon and menu bar icon needs to have links (a-tags)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,805 @@
...
+        </nav>
+
+    <!--Content Section -->
+
+    <section>
+        <div class="banner-grid">
+            <div class="container opal">
+                <p class="right">Tuesday — January 26, 2021</p>
+                <img src="images/tnw.svg" class="tnw-svg" alt="tnw-pic">                
+                <h1>Opal cuts off your apps’ internet access to help you stop doomscrolling</h1>
```
Comment (henatan99) [inline index.html]: This h1-element should also be linked (add a-tag). This comment applies to the other two articles in this section (container google and container mac).

File: index.html [hunk 8]
```diff
@@ -0,0 +1,805 @@
...
+                </div>
+            </div>
+    </div>    
+
+
+    <!--More articles Section -->
+
+    <div class="container-fluid">
+        <a href="#"> <h1 class="orange-title"> Latest funding rounds on Index.co ></h1> </a>
+        <div class="latest-fund-links">
```
Comment (henatan99) [inline index.html]: Use nav tag instead of div here

File: index.html [hunk 9]
```diff
@@ -0,0 +1,805 @@
...
+        <a href="#"> <p class="small-title">See more </p> </a> <p class="invisible1">|</p>
+        <a href="#"> <p class="small-title">Get updates by email </p> </a> <p class="invisible1">|</p>
+        <a href="#"> <p class="small-title">Add info </p> </a>
+        </div>
+    </div>
+<div class="red-container">
+    <div class="container ring">
+        <div class="red-box">
+            <h2>Ring Over</h2>
```
Comment (henatan99) [inline index.html]: The title of each article in the latest funding section should have a-rag. Extrapolate this comment to the other articles too

File: index.html [hunk 10]
```diff
@@ -0,0 +1,805 @@
...
+            <img class= "ring-over" src="Images/ring-over.JPG" alt="ring-over">
+    </div>
+    <div class="light-red">
+        <h2>$12M</h2>
+        <p>SERIES A</p>
+    </div>
+    <div class="white">
+        <p><i class="fa fa-tag" aria-hidden="true"></i>
+            &nbsp;Telecommunications</p>
+        <p><i class="fa fa-briefcase" aria-hidden="true"></i>
```
Comment (henatan99) [inline index.html]: These elements should have links (a-tag). Apply the same change for the other articles too

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes required :recycle:

Hi there, you have done an amazing job so far. :clap: :clap: :clap: But you need so make a few more changes to fulfil all the project's requirements. Please go through the review and make the necessary changes. 

### Reminder: It is your first review and you will have a maximum of two more reviews (re-reviews), so please make all the requested changes before you submit for another CRR.

## To Highlight 
- The original design is well cloned 
- Nice responsive design 
- Nice typography, color contrast and spacing 
- Amazing readme 

## To Improve [MANDATORY]
- It is a requirement to follow a mobile-first design approach, the base styling should be for mobile and apply media queries for tablet and desktop views. But you did desktop-first approach. 
- Unnecessary article tag is used as a container inside the body, please see the inline comment and make the necessary change 
- The nav elements should have a proper nav tag and wrapped inside a header tag, please see the inline comment and make changes accordingly 
- Both menu bar icon and search icon need to be linked (a-tag). See inline comment 
- The highlights section elements (container opal, container google, and container mac) should have an article tag instead of div. Also, the titles in these articles should be linked (adds a-tags). See inline comments. 
- Latest funding menus should be wrapped in a nav tag. See inline comment 
- The latest section title and menu are not aligned with the articles in mobile view

### PLease make the requested changes and submit for another review.

### Reminder: It is your first review and you will have a maximum of two more reviews (re-reviews), so please make all the requested changes before you submit for another CRR. If your project doesn't get approved after the third review, you will repeat the project.

### Happy coding :computer: :computer_mouse:

File: index.html [hunk 11]
```diff
@@ -0,0 +1,802 @@
...
+            </nav>
+        </header>
+
+    <!--Content Section -->
+
+    <section>
+        <div class="banner-grid">
+            <div class="container opal">
+                <p class="right">Tuesday — January 26, 2021</p>
+                <img src="images/tnw.svg" class="tnw-svg" alt="tnw-pic">
```
Comment (christianotieno) [inline index.html]: ## `tnw` image not showing
- [x] The `tnw` logo is currently not loading. Do you mind fixing this?
See the comparison below 👇🏾 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/mariordgez/Responsive_Design_Habeeb_Mario/Milestone_1/index.html)|
|:--:|:--:|
|source|clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @mariordgez  & @Edungit 👋🏾 
Thank you for working on the majority of the changes. keep it up 👏🏾  
Below there is a suggestion on the `Main article` section that might need our attention. have a look 👇🏾 

> **"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second opinion using this form. ](https://airtable.com/shrQAqnBwek5a0O0s)"**

### Happy coding 💻 
#### Christian Otieno  (CR)

**Comment by mariordgez** [comment]
If img doesnt load on firefox, use open with a private window
imagen
I hope you like the project, have a nice day!.

**Review by christianotieno — APPROVED** [review]
## Project Approved 🍾 
Thank you for working on the required changes.👍🏾 
Congratulations. The project is now approved. 👏🏾 
All the very best in your next tasks 🥇 
### Happy coding 💻 
#### Christian Otieno (CR)

**Pull request merged**